src/app/pages/wallets/wallet-details/pages-wallet-details-item.model.ts

```diff
@@ -0,0 +1,15 @@
+import { IWalletTransactionApiResponse } from 'src/app/domains/wallets/domains.wallets.types';
+
+export class WalletsDetailsTransaction {
+  public readonly id: number | null;
+  public readonly date: Date;
+  public readonly description: string;
+  public readonly amount: number;
+
+  public constructor(data: IWalletTransactionApiResponse) {
+    this.id = data.id === 0 ? null : data.id;
+    this.date = new Date(data.creationDate);
+    this.description = data.description;
```

Comment:
Description is an optional property so you need to:
- adjust type of the API response so `description` is optional
- add fallback here -> so when description is missing or null we should fallback to null value